### PR TITLE
Feature/percent change

### PR DESCRIPTION
**Pull request by jaredledvina**
Adds the percent changed from open.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -67,8 +67,9 @@ func (c *SeabirdClient) stockCallback(event *pb.CommandEvent) {
 		} else {
 			company = fmt.Sprintf("%s", ticker)
 		}
+		percentChange := ((quote.O - quote.C) / quote.O) * 100
 		// TODO: Don't hardcoded USD here - currency requires premium https://finnhub.io/docs/api#company-profile
-		c.Replyf(event.Source, "%s: %s - Open: $%.2f, Current: $%.2f", event.Source.GetUser().GetDisplayName(), company, quote.O, quote.C)
+		c.Replyf(event.Source, "%s: %s - Open: $%.2f, Current: $%.2f (%.2f %%)", event.Source.GetUser().GetDisplayName(), company, quote.O, quote.C, percentChange)
```
Comment (belak) [inline client.go:72]: This looks good, but do we want do make sure there's a sign? Like `+42.0%` as well as `-42%`?

Comment (jaredledvina) [reply client.go:72]: Oh, I thought that's what the `%%` would do @belak

Comment (belak) [reply client.go:72]: I was talking about making sure there was a sign, I think the `+` flag does that.

https://play.golang.org/p/CB4mXAydFdY

Comment (jaredledvina) [reply client.go]: Ah gotcha, done!

**Pull request merged**